lib/resultsCapturer.js

```diff
@@ -245,8 +253,8 @@ function commonStepFinished({
         scenarioFailed = true;
         stepResults.assertions.push({
           passed: false,
-          errorMsg: exception.message,
-          stackTrace: exception.stack
+          errorMsg: exception ? exception.message : 'No Exception message found',
```

Comment:
I think `exception?.message || 'Ambigous step found` will be better. It also handles the case where message is undefined or empty.